### PR TITLE
Updating Template

### PR DESCRIPTION
**Pull request by Exactor96**

### CODE REVIEW

**Comment by paul-pro** [comment]
Hey team, I need a review. Mainly from @henderjarr 
cc @Sc4ramouche @jrvaja @ignatvilesov

**Review by henderjarr — APPROVED** [review]
yeah, this looks solid, need to test it though.

File: results/reports/mochawesome.json [hunk 1]
```diff
@@ -0,0 +1,130 @@
+{
```
Comment (paul-pro) [inline results/reports/mochawesome.json:1]: @Exactor96, do we need this file? Should there be a .gitignore setting to ignore all reports?

Comment (paul-pro) [reply results/reports/mochawesome.json:1]: and videos

**Review by paul-pro — APPROVED** [review]
Everything else LGTM, let's just remove reports and videos from the repo

File: results/reports/mochawesome.json (hunk 1 above)
Comment (Exactor96) [reply results/reports/mochawesome.json]: Done.

**Pull request merged**